server/controller/textScan.controller.js

```diff
@@ -48,6 +48,8 @@ textScanController.post(
         semesters: semesters
```

Comment:
The Plan schema also includes a field 'program' which contains an ID to a degree program. Shouldn't we also add a 'program' field here?